cypress/integration/content.js

```diff
@@ -39,6 +39,28 @@ describe('Add Content Tests', () => {
       cy.contains('This is the text');
     }
   });
+  it('As a site administrator I can add a new line to a text block using shift + enter', function() {
```

Comment:
@rodfersou I think this test should go into block-text.js. This is where we have all the Text Block related tests.